spec/src/main/asciidoc/app-programming-model.adoc

```diff
@@ -190,6 +192,9 @@ The implementation must support the following annotation targets:
   * `METHOD`
   * `TYPE`
 
+WARNING: `@Counted` is deprecated, please use `@HitCounted` / `@ParallelCounted`.
+`@Counted` will be removed in MicroProfile-Metrics 3.0.
```

Comment:
It seems unusual to say exactly when something will be removed (in particular it also presupposes that there will be a release called 3.0, which is likely, but not something we should necessarily say in a spec.)